finch.yaml

```diff
@@ -148,27 +148,28 @@ provision:
 - mode: user
   script: |
     #!/bin/bash
-    sudo chown $USER /mnt/lima-finch
 
-    # https://github.com/containerd/nerdctl/blob/cffdf87ff4d648a5344eea1406bb95ca3ad7eaa4/extras/rootless/containerd-rootless.sh#L144-L146
-    # XDG_DATA_HOME & ~/.local/share: https://github.com/containerd/nerdctl/blob/cffdf87ff4d648a5344eea1406bb95ca3ad7eaa4/extras/rootless/containerd-rootless.sh#L51
-    mkdir -p /mnt/lima-finch/containerd ~/.local/share/containerd
-    sudo mount --bind /mnt/lima-finch/containerd ~/.local/share/containerd
+    # Enable SSHing into the VM as root (e.g., in `nerdctlConfigApplier.Apply`).
+    sudo cp ~/.ssh/authorized_keys /root/.ssh/
+    sudo chown $USER /mnt/lima-finch
 
     # https://github.com/containerd/nerdctl/blob/main/docs/dir.md#dataroot
-    mkdir -p /mnt/lima-finch/nerdctl ~/.local/share/nerdctl
-    sudo mount --bind /mnt/lima-finch/nerdctl ~/.local/share/nerdctl
+    sudo mkdir -p /mnt/lima-finch/nerdctl /var/lib/nerdctl
+    sudo mount --bind /mnt/lima-finch/nerdctl /var/lib/nerdctl
 
     # https://github.com/containerd/nerdctl/blob/main/docs/dir.md#netconfpath
-    mkdir -p /mnt/lima-finch/cni-config ~/.config/cni
-    sudo mount --bind /mnt/lima-finch/cni-config ~/.config/cni
+    sudo mkdir -p /mnt/lima-finch/cni-config /etc/cni/
+    sudo mount --bind /mnt/lima-finch/cni-config /etc/cni/
+
+    # https://github.com/containerd/containerd/blob/main/docs/ops.md#base-configuration
+    sudo mkdir -p /var/lib/containerd
+    sudo mount --bind /mnt/lima-finch/containerd /var/lib/containerd
```

Comment:
nit: move this block to be where the `~/.local/share/containerd` part was (i.e., right after `sudo chown $USER /mnt/lima-finch`) may make it easier to review.